nucypher/config/keyring.py

```diff
@@ -249,54 +258,54 @@ def _generate_tls_keys(host: str, checksum_address: str, curve: EllipticCurve) -
     return private_key, cert
 
 
-class _PrivateKeySerializer:
+class _TLSPrivateKeySerializer:
```

Comment:
Wile I do like the encapsulation and usage of namespaces here this class only has two static methods, consider moving it to a module with a similar name, or leaving it as functions.  Good to see some unit tests for this area :+1: